### ISSUE TITLE
Please please please complete your documentation.

### ISSUE THREAD

**Issue by tpst** [JoshClose/CsvHelper#1898]
Every year I download this project to use and every year I hit some asenine problem, open the documentation and a page that still says  [https://joshclose.github.io/CsvHelper/examples/type-conversion/]("Coming soon.."). 

This is such a fantastic project but the lack of documentation is extremely frustrating, especially for something so popular. 

Why no list of classes, methods & properties?

I'm sure many others would appreciate a complete & fleshed out documentation for this project.

**Comment by JoshClose** [comment]
I agree. I should finish out the documentation before working on any more features.

> Why no list of classes, methods & properties?

I switched the static html tool I'm using from react-static to [Statiq.Web](https://www.statiq.dev/web/). I was in the process of building out a generator for that stuff, but the Statiq devs said they're working on and close to being done with a generator. Once that is complete (even in a beta version), I'll add that to the site.

For now, in Visual Studio you can use `View -> Object Browser` and see everything that way.

**Comment by mariusgaus** [comment]
Any Updates?

**Comment by JoshClose** [comment]
Looks like the docs part might be available now... https://www.statiq.dev/docs I'll check it out and if it is, I can push that.
